src/avatar.jsx

```diff
@@ -209,100 +212,112 @@ class Avatar extends React.Component {
-      const y = isTopCorner() ? calcTop() : (isBottomCorner() ? calcBottom() : crop.y())
-      moveResizer(x, y)
-      crop.setFillPatternOffset({ x: x / this.scale, y: y / this.scale })
-      crop.x(x)
-      cropStroke.x(x)
-      crop.y(y)
+      const x = isLeftCorner() ? calcLeft() : (isRightCorner() ? calcRight() : crop.x());
+      const y = isTopCorner() ? calcTop() : (isBottomCorner() ? calcBottom() : crop.y());
+      moveResizer(x, y);
+      crop.setFillPatternOffset({ x: x / this.scale, y: y / this.scale });
+      crop.x(x);
+      cropStroke.x(x);
+      crop.y(y);
       cropStroke.y(y)
-    })
+    });
 
-    crop.on("mouseenter", () => stage.container().style.cursor = 'move')
-    crop.on("mouseleave", () => stage.container().style.cursor = 'default')
-    crop.on('dragstart', () => stage.container().style.cursor = 'move')
-    crop.on('dragend', () => stage.container().style.cursor = 'default')
+    crop.on("mouseenter", () => stage.container().style.cursor = 'move');
+    crop.on("mouseleave", () => stage.container().style.cursor = 'default');
+    crop.on('dragstart', () => stage.container().style.cursor = 'move');
+    crop.on('dragend', () => stage.container().style.cursor = 'default');
 
     resize.on("touchstart", (evt) => {
       resize.on("dragmove", (dragEvt) => {
-        if (dragEvt.evt.type !== 'touchmove') return
-        const scaleY =  (dragEvt.evt.changedTouches['0'].pageY - evt.evt.changedTouches['0'].pageY) || 0
+        if (dragEvt.evt.type !== 'touchmove') return;
+        const scaleY = (dragEvt.evt.changedTouches['0'].pageY - evt.evt.changedTouches['0'].pageY) || 0;
         onScaleCallback(scaleY * this.mobileScaleSpeed)
       })
-    })
+    });
 
     resize.on("dragmove", (evt) => {
-      if (evt.evt.type === 'touchmove') return
-      const scaleY = evt.evt.movementY || 0
+      if (evt.evt.type === 'touchmove') return;
+      const newMouseY = evt.evt.y;
+      const ieScaleFactor = newMouseY ? (newMouseY - this.state.lastMouseY) : undefined;
+      const scaleY =  evt.evt.movementY || ieScaleFactor || 0;
+  	  this.setState({
+  		  lastMouseY: newMouseY,
+  	  });
       onScaleCallback(scaleY)
-    })
-    resize.on("dragend", () => this.onCropCallback(getPreview()))
-
-    resize.on('resize', () => moveResizer(crop.x(), crop.y()))
-
-    resize.on("mouseenter", () => stage.container().style.cursor = 'nesw-resize')
-    resize.on("mouseleave", () => stage.container().style.cursor = 'default')
-    resize.on('dragstart', () => stage.container().style.cursor = 'nesw-resize')
+    });
+    resize.on("dragend", () => this.onCropCallback(getPreview()));
+
+    resize.on('resize', () => moveResizer(crop.x(), crop.y()));
+
+    resize.on("mouseenter", () => stage.container().style.cursor = 'nesw-resize');
+    resize.on("mouseleave", () => stage.container().style.cursor = 'default');
+    resize.on('dragstart', (evt) => {
+      console.log('resize dragStart');
```

Comment:
@Seppo007 did you forgot to remove it ?

Reply:
yeah, it seems that I forgot to remove it ;)

Reply:
@kirill3333 has been removed with new commits